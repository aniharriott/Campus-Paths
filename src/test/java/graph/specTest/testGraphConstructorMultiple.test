# Test Graph constructor multiple file.
# Tests if creating a multiple graphs
# at a time works.

# Create graphs
CreateGraph graph1
CreateGraph graph2