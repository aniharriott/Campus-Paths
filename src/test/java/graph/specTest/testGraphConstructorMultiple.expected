# Test Graph constructor multiple file.
# Tests if creating a multiple graphs
# at a time works.

# Create graphs
created graph graph1
created graph graph2